src/cursor/cur_layered.c

```diff
@@ -236,27 +235,31 @@ __clayered_open_cursors(WT_CURSOR_LAYERED *clayered, bool update)
 
     F_CLR(clayered, WT_CLAYERED_ITERATE_NEXT | WT_CLAYERED_ITERATE_PREV);
 
-    /* Always open the ingest cursor */
+    /* Always open both the ingest and stable cursors */
     if (clayered->ingest_cursor == NULL) {
         WT_RET(__wt_open_cursor(
           session, layered->ingest_uri, &clayered->iface, NULL, &clayered->ingest_cursor));
         F_SET(clayered->ingest_cursor, WT_CURSTD_OVERWRITE | WT_CURSTD_RAW);
     }
 
-    if (clayered->stable_cursor == NULL && F_ISSET(clayered, WT_CLAYERED_OPEN_READ)) {
-        ckpt_cfg[0] = WT_CONFIG_BASE(session, WT_SESSION_open_cursor);
-        ckpt_cfg[1] = "checkpoint=" WT_CHECKPOINT ",raw,checkpoint_use_history=false";
-        ckpt_cfg[2] = NULL;
+    if (clayered->stable_cursor == NULL) {
+        if (!S2C(session)->layered_table_manager.leader) {
```

Comment:
Do you know why on the follower we need to open a checkpoint?

Reply:
Oops, I was experimenting and left this in. Fixed in `cd7de97`.